### PR TITLE
List tag of package installed with --version

### PR DESCRIPTION
**Pull request by goodlyrottenapple**
Fixes https://github.com/runtimeverification/kup/issues/41

image

### CODE REVIEW

File: src/kup/__main__.py [hunk 1]
```diff
@@ -336,7 +342,7 @@ def list_package(package_name: str, show_inputs: bool) -> None:
     else:
         table_data = [
             ['Package', 'Installed version', 'Status'],
-        ] + [[name, p.version, p.status] for name, p in packages.items()]
+        ] + [[name, f'{p.version}{" (" + p.tag + ")" if p.tag else ""}', p.status] for name, p in packages.items()]
```
Comment (tothtamas28) [inline src/kup/__main__.py:345]: Consider extracting this expression:
```python3
tag_str = f' ({p.tag})' if p.tag else ''
```

then

```python3
f'{p.version}{tag_str}'
```

File: src/kup/__main__.py [hunk 2]
```diff
@@ -229,16 +229,21 @@ def reload_packages(load_versions: bool = True) -> None:
...
-                        len(m['originalUrl'].removeprefix(f'github:{available_package.org}/{available_package.repo}'))
-                        > 1
+                    maybe_tag = m['originalUrl'].removeprefix(
+                        f'github:{available_package.org}/{available_package.repo}'
                     )
+                    if len(maybe_tag) > 1:
+                        immutable = True
+                        tag = maybe_tag.removeprefix('/')
+                    else:
+                        immutable = False
```
Comment (tothtamas28) [inline src/kup/__main__.py:246]: Consider bringing the assignment above to a more local context:

```suggestion
                    tag: Optional[str]
                    if len(maybe_tag) > 1:
                        immutable = True
                        tag = maybe_tag.removeprefix('/')
                    else:
                        immutable = False
                        tag = None
```

**Review by tothtamas28 — APPROVED** [review]
Looks good!

File: src/kup/__main__.py (hunk 1 above)
Comment (goodlyrottenapple) [reply src/kup/__main__.py:345]: its inside a list comprehension, can i do a local binding there?

**Pull request merged**